artemis/src/main/java/tech/pegasys/artemis/DepositCommand.java

```diff
@@ -80,45 +100,98 @@ public void generate(
               defaultValue = "1")
           int validatorCount,
       @Option(
-              names = {"--output-file", "-o"},
+              names = {"--output-path", "-o"},
+              paramLabel = "<FILE|DIR>",
               description =
-                  "File to write validator keys to. Keys are printed to std out if not specified")
-          String outputFile) {
+                  "Path to output file for unencrypted keys or output directory for encrypted keystore files. If not set, unencrypted keys will be written on standard out and encrypted keystores will be created in current directory")
+          String outputPath,
+      @Option(
+              names = {"--encrypt-keys", "-e"},
+              defaultValue = "true",
+              paramLabel = "<true|false>",
+              description = "Encrypt validator and withdrawal keys. (Default: true)",
+              arity = "1")
+          boolean encryptKeys,
+      @Option(
+              names = {"--validator-password:file"},
+              paramLabel = "<FILE>",
+              description = "Read password from the file to encrypt the validator keys")
+          File validatorPasswordFile,
+      @Option(
+              names = {"--validator-password:env"},
+              paramLabel = "<ENV_VAR>",
+              description = "Read password from environment variable to encrypt the validator keys")
+          String validatorPasswordEnv,
+      @Option(
+              names = {"--validator-password"},
```

Comment:
It seems weird to have to specify `--validator-password` to get the prompt to provide the password. Makes me naturally want to specify the password on the command line which is insecure and doesn't actually work.